src/main/java/pl/vishop/vishopplugin/config/Config.java

```diff
@@ -46,8 +54,42 @@ public boolean load(final ConfigurationSection cfg) {
             return false;
         }
 
+        this.broadcastMessages = new HashMap<>();
+        ConfigurationSection broadcastSection = cfg.getConfigurationSection("broadcasts");
+        boolean isSpigot = false;
+        try {
+            Class.forName("net.md_5.bungee.api.ChatColor");
+            isSpigot = true;
+        } catch (ClassNotFoundException ignored) {}
+
+        boolean translateHex = isSpigot && (Bukkit.getServer().getVersion().contains("1.16") || Bukkit.getServer().getVersion().contains("1.17") || Bukkit.getServer().getVersion().contains("1.18") || Bukkit.getServer().getVersion().contains("1.19"));
+        broadcastSection.getKeys(false).forEach(id -> {
+            List<String> messages = broadcastSection.getStringList(id);
+            int i = 0;
+            for (String msg : messages) {
+                msg = ChatColor.translateAlternateColorCodes('&', msg);
+                if (translateHex) msg = translateHex(msg);
+                messages.set(i, msg);
```

Comment:
Nie wiem czemu set a nie add - można by to całe zrobić kompletnie funkcyjnie mapowaniami odpowiednimi